docs/theming.md

```diff
@@ -1,113 +1,209 @@
-# Theming your Alfresco ADF app
+# Theming an ADF app
+
+The [Material Design](https://material.io/guidelines/material-design/introduction.html)
+language doesn't specify a single color scheme. Instead it uses the concept
```

Comment:
I guess more than call it language we should refer it as specification or guidelines